docs/miscellaneous/cli.md

```diff
@@ -58,6 +59,19 @@ Flags:
   -h, --help         help for refresh
 ```
 
+## export
+
+```text
+export an API
+
+Usage:
+  cortex export API_NAME [flags]
+
+Flags:
+  -e, --env string   environment to use (default "local")
```

Comment:
the default should be "aws" (also update the code if necessary to ensure that's the behavior)